sdk/core/Azure.Core/src/Internal/RehydrationOperationOfT.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
```

Comment:
Please enable nullable annotations for types in Azure.Core.

Reply:
enabled